### PR TITLE
Pash -c --command

### PR DESCRIPTION
**Pull request by dkarnikis**

### CODE REVIEW

File: compiler/pash.py [hunk 1]
```diff
@@ -74,6 +77,21 @@ def parse_args():
                         action="store_true")
     config.add_common_arguments(parser)
     args = parser.parse_args()
+    # handle the script argument as executable code
+#    if args.input and args.command:
```
Comment (angelhof) [inline compiler/pash.py:81]: Is this code dead? If so we could remove it :)

**Review by angelhof — APPROVED** [review]
Did you run the tests in `compiler/test_evaluation_scripts.sh`? If so we can merge after the commented out code is removed.

**Comment by nvasilakis** [comment]
@angelhof This is waiting for some changes from me.

**Comment by nvasilakis** [comment]
Cherry-picked 38db981d9c54da1f3cba37651397870aa304dacd in #143

**Pull request closed without merging**